libs/sdk-ui-dashboard/src/presentation/dashboard/types.ts

```diff
@@ -578,6 +578,29 @@ export interface IDashboardEventing {
     onStateChange?: (state: DashboardState, dispatch: DashboardDispatch) => void;
 }
 
+/**
+ * @internal
+ */
+export enum DashboardPartIds {
+    MainContent = "mainContent",
+    FiltersBar = "filtersBar",
+    Sidebar = "sidebar",
+}
+
+/**
+ * @internal
+ */
+export interface INavigationConfigItem {
+    tabIndex?: number;
+    navigationId?: string;
+    onFocus?: () => void;
+}
+
+/**
+ * @internal
+ */
+export type NavigationConfig = Record<DashboardPartIds, INavigationConfigItem>;
```

Comment:
Would it make sense to name it `KeyboardNavigation` to convey what sorts of navigation are we talking about? Subsequently you could change the prop to `keyboardNavigation` below.